README.md

```diff
@@ -2,6 +2,17 @@
 
 REST server that indexes search documents within an Atlas cluster and handles user search requests, deployed via Kanopy
 
+## Components
+
+### Marian Server
+REST server that exposes endpoints to run queries against A DB.
+
+### Atlas Admin Manager
+Module to invoke Atlas Admin commands via the Atlas Admin API.
```

Comment:
(Non-blocking suggestion) Would it be a good idea to include a sentence to provide an overview/more context on what the purpose of these commands are for? Something like `"The module syncs the Search Index to ensure content taxonomy is incorporated in faceted search data."` ?

Reply:
good suggestion! i didn't want to explain specific details of projects so i added the admin API, as well as the Search Index. lmk if this works!